ios/objc/testApp/objc/TestApp/TestApp/utils/ImagePredictor.mm

```diff
@@ -0,0 +1,69 @@
+#import "ImagePredictor.h"
+#import <torch/script.h>
+#import <PytorchObjC/PytorchObjC.h>
+#import <vector>
+#import "UIImage+Utils.h"
+
+#define IMG_W 224
+#define IMG_H 224
+#define IMG_C 3
+
+@implementation ImagePredictor {
+    TorchModule* _module;
+    NSArray* _labels;
+}
+
+- (instancetype)initWithModelPath:(NSString* )modelPath {
+    self = [super init];
+    if (self) {
+        _module = [TorchModule loadTorchscriptModel:modelPath];
```

Comment:
Ideally, this should happen on a background thread, since it accesses disk and can take quite some time, blocking the main thread. Which means we'd better have a separate method with a completion handler, like `loadModelAtPath:completion:` to do this work.
Same concerns the `stringWithContentsOfFile` below, it should also happen on the background thread together with `load`.